util/all_tests.go

```diff
@@ -95,6 +85,40 @@ var sdeCPUs = []string{
 	"cpx", // Cooper Lake
 	"icx", // Ice Lake server
 	"tgl", // Tiger Lake
+	"snb", // Sandy Bridge
+	"ivb", // Ivy Bridge
+}
+
+// CPUs that are NOT compatible with MSVC
+var cpusWithoutAVX = []string{
```

Comment:
Comment and name seem out of date. The root cause is really modern MSVC assumes a minimum level of CPU support.